CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 - Migrate `BloodSugarEntrySheet` to use `BaseBottomSheet`
 - Migrate `UpdatePhoneNumberDialog` to `BaseDialog`
 - Bump dagger to v2.43.2
-- Bump gradle to v7.5.1
+- Bump AGP to v7.2.2
```

Comment:
I meant as a separate line. We have update Gradle and AGP as well. 

Gradle is updated to v7.5.1
AGP is updated to v7.2.2